docs/tables/rdap_domain.md

```diff
@@ -0,0 +1,194 @@
+# Table: rdap_domain
```

Comment:
@ParthaI Can you please update this table doc so it matches our latest doc format (if it doesn't already)?

Reply:
@cbruno10, I have updated the dos to match our latest doc format.